cmd/ecspresso/main.go

```diff
@@ -21,7 +21,7 @@ func main() {
 func _main() int {
 	kingpin.Command("version", "show version")
 
-	conf := kingpin.Flag("config", "config file").String()
+	conf := kingpin.Flag("config", "config file").Default("ecspresso.yml").String()
```

Comment:
This value is `config.yaml` in existing documents, but I think `ecspresso.yml` is better because more identical, especially as a default value.

But it is just my opinion. How about this?

Reply:
That's right. I'll fix the documents when the next version is released.